lib/taurus/qt/qtgui/__init__.py

```diff
@@ -54,16 +55,34 @@
 # Note: this is an experimental feature introduced in v 4.3.0a
 # It may be removed or changed in future releases
 
+class LazyModule(ModuleType):
+
+    def __init__(self, name, package, entry_point):
+        super(LazyModule, self).__init__(name)
+        self.__package__ = package
+        self.ep = entry_point
+
+    def __getattr__(self, member):
+        import sys as __sys
```

Comment:
why is this import (and later deletion) of `sys` needed? 
AFAIK, `sys` is already imported at global module level...

Reply:
I think this is called later, when `sys` is already deleted. However, I'm not sure about that. Python complains about not founding module when there's not import

Reply:
Oh... I see... it is because of the last line in the __init__ that is cleaning the namespace

I do  not remember why exactly we bothered in cleaning the namespace in the case of the __init__. Maybe it had something to do with the autodocumentation...